src/resources/authoring/gv2.json

```diff
@@ -426,6 +398,57 @@
         },
         "visibleGenes": "metallic, wings, forelimbs, hindlimbs"
       }
+  },
+
+
+  "levelHierarchy": [
+    // Level 1
```

Comment:
We should have valid JSON. It's potentially another argument for having objects which could then have additional fields for `comments`, `description`, etc. It could look something like:
```
{
  challenges: {
    ...
  },
  application: {
    "name": "...",
    "description": "...",
    "levels": [
      {
        "name": "Level 1",
        "description": "...",
        "missions": [
          {
            "name": "Mission 1",
            "description": "...",
            "challenges": [
              {
                "name": "Challenge 1",
                "location": "Hatchery",
                "challengeId": "someUniqueChallengeID"
              },
              {
                "name": "Challenge 2",
                "location": "Hatchery",
                "challengeId": "anotherUniqueChallengeID"
              }
            ]
          },
          {
            "name": "Mission 2",
            "description": "...",
            "challenges": [
              {
                "name": "Challenge 1",
                "location": "Hatchery",
                "challengeId": "yetAnotherUniqueChallengeID"
              },
              {
                "name": "Challenge 2",
                "location": "Hatchery",
                "challengeId": "stillAnotherUniqueChallengeID"
              }
            ]
          }
        ]
      },
      {
        "name": "Level 2",
        "missions": []
      }
    ]
  }
}
```